src/site/layouts/page.drupal.liquid

```diff
@@ -26,6 +26,10 @@
         {% endif %}
 
         <article class="usa-content">
+          <div id="i18-link-wrapper" class="vads-u-margin-y--3 medium-screen:vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--3">
+            <i class="fas fa-globe vads-u-color--primary"></i> 
+            <a class="i18-toggle" data-lang="en"></a>
```

Comment:
What's this for? If we need to link it will need some content, even if it is hidden from a screen reader. Also, links without href values announce as disabled, is that the intention here?

Reply:
the content of the link is set dynamically here: https://github.com/department-of-veterans-affairs/vets-website/pull/15289/files#diff-87cfcd77dc2ca13698eebe19ce454bf18ca6c555f757e0759fbcf9155d02ced4R24
I can set an initial value and hide it from screen reader / set to display none. the href is also set on dom content loaded as well, so it wouldn't be disabled once dom content has loaded.

Reply:
ok, that sounds fine